Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/invocation/SkipInvocation.kt

```diff
@@ -0,0 +1,16 @@
+package org.cqframework.cql.cql2elm.model.invocation
+
+import org.hl7.elm.r1.Expression
+import org.hl7.elm.r1.Slice
+
+/** Created by Bryn on 5/17/2017. */
```

Comment:
I wonder if this comment is misleading, since this is technically a brand-new Kotlin class.

Reply:
I debated that. It's kind of a low-value comment. But Bryn is still the #1 expert on the compiler design so I decided that I'd rather people go to him with questions on the class based on the comment than to me based on the commit log. :)